internal/controller/challenges/apply.go

```diff
@@ -63,7 +63,7 @@ func ApplyStatus(ctx context.Context, cl cmclient.Interface, fieldManager string
 }
 
 // serializeApply converts the given Challenge object in JSON. Only the
-// objectmeta, and spec fields will be copied and encoded into the serialized
+// ObjectMeta, and Spec fields will be copied and encoded into the serialized
```

Comment:
I was going to let this go until I realized that `TypeMeta` is written in CamelCase two lines below...